0.download_data/scripts/nbconverted/download_data.py

```diff
@@ -10,38 +10,57 @@
 import pandas as pd
 
 
-# ### Specify version of mitocheck_data to download from
+# ## Specify mitocheck_data types and paths
+# 
+# We use multiple types of data for model training/testing.
 
 # In[2]:
 
 
-labeled_data_hash = "20369033f579dca1334cb2c58a1c6d532322f93e"
-labeled_data_url = f"https://raw.github.com/WayScience/mitocheck_data/{labeled_data_hash}/3.normalize_data/normalized_data/training_data.csv.gz"
+labeled_data_hash__ic = "20369033f579dca1334cb2c58a1c6d532322f93e"
+labeled_data_url__ic = f"https://raw.github.com/WayScience/mitocheck_data/{labeled_data_hash__ic}/3.normalize_data/normalized_data/training_data.csv.gz"
 
+# ADD PATH TO DATA FROM GITHUB
+# labeled_data_hash = ""
+labeled_data_url__no_ic = f"/home/roshankern/Desktop/Github/mitocheck_data/3.normalize_data/normalized_data__no_ic/training_data.csv.gz"
```

Comment:
consider  changing to a relative path and resolving the absolute path via 
`pathlib.Path(f"{realtive_path}").resolve()`

This so others can use this code reproducibly who on other workstations that do not have a `roshankern` user

Reply:
I will leave this as is so it is clear where to change later. Unfortunately we are developing the PRs a little out of order, but this will be fixed before merging into main!